src/test/suite/asciidocUtilsTest.test.ts

```diff
@@ -42,4 +49,16 @@ suite('AsciiDoc Utils Test Suite', () => {
 		expect(content).to.include('The quick brown fox jumps over the lazy dog.');
 	});
 
+	test('get the adoc parser', () => {
+		expect(adocutils.getADParser()).to.not.be.null;
+	});
+
+	test('that we can parse adoc to html', () => {
+		const htmlOutput = adocutils.parseADtoHTML(BASIC_EXAMPLE);
+		expect(htmlOutput).to.include(`<p>That&#8217;s all, folks!</p>`);
+
+		// this is tested more effectively with 'open an asciidoc file with an include' above
+		const htmlOutputWithBaseDir = adocutils.parseADtoHTML(BASIC_EXAMPLE, '');
+		expect(htmlOutputWithBaseDir).to.include(`<p>That&#8217;s all, folks!</p>`);
+	});
```

Comment:
what's the link with the Sonar reported issues?

Reply:
"Not covered by tests" - https://sonarcloud.io/code?id=vscode-didact&selected=vscode-didact%3Asrc%2FasciidocUtils.ts

Reply:
spinning off to https://github.com/redhat-developer/vscode-didact/pull/293